horton/gbasis/common.cpp

```diff
@@ -150,3 +152,82 @@ void nuclear_attraction_helper(double* work_g, long n0, long n1, double pa, doub
         work_g[index] = tmp;
     }
 }
+
+
+/*
+
+    Auxiliary functions for r^alpha integrals
+
+*/
+
+
+double cit(int i, double t, int m) {
+  double result = pow(t, m);
+  for (int j=1; j <= i; j++)
+    result *= 4.0/((2.0*j + 1.0)*(2.0*j));
+  return result;
+}
+
+
+long jfac(int j, int n) {
+  double result = j;
+  for (int i=1; i < n; i++)
+    result *= j-i;
+  return result;
+}
+
+
+double dtaylor(int n, double alpha, double t, double prefac) {
```

Comment:
Here in my branch I have:
    /* This function evaluated the taylor series for r^alpha integrals
    Sum_i t^i * Gamma(i+(alpha+3)/2)*(2^(2i)/(2i +1)!

    Arguments:
        n : the angular moment (the order of the derivative of the basic integral Gn in
        Alhrichs Phys. Chem. Chem. Phys., 8, 3072 (2006))
        the maximum value implemented is n=10.
        t : rho|p-q|^2
        prefac : e^(-t)/rho^3/2 - this term helps the Taylor series to converge when t is a large number,
             the factor 1/2Sqrt(rho^alpha) was "replaced" and multiplied outside, at the end, in the
             laplace_of_potential function.

   */